Cargo.Bazel.toml.lock

```diff
@@ -306,7 +306,7 @@ dependencies = [
  "serde",
  "serde_json",
  "serde_qs",
- "thiserror",
+ "thiserror 1.0.68",
```

Comment:
looks like we're now using 2 different versions of `thiserror`, is this expected?

Reply:
what's the reason for keep the old version around? I think the PR title is a bit misleading; this does not exactly upgrade `thiserror` but instead introduces a new version